### PR TITLE
fix: update to use @ttab/elephant-api

### PR DESCRIPTION
**Pull request by gularsson**
Use generated typescript definitions from protos from a npm package... ~waiting for package build~

### CODE REVIEW

File: src-srv/utils/CollaborationServer.ts [hunk 1]
```diff
@@ -305,10 +305,12 @@ export class CollaborationServer {
     }
 
     // Fetch content
+    console.log('::: uuid', uuid)
```
Comment (dannelundqvist) [inline src-srv/utils/CollaborationServer.ts:308]: Maybe use `console.debug()` instead.

File: src-srv/utils/CollaborationServer.ts [hunk 2]
```diff
@@ -305,10 +305,12 @@ export class CollaborationServer {
...
     const newsDoc = await this.#repository.getDoc({
       uuid,
       accessToken: context.accessToken
     }).catch(ex => {
+      console.log('Error fetching document', ex)
```
Comment (dannelundqvist) [inline src-srv/utils/CollaborationServer.ts:313]: Maybe use console.error() instead.

Comment (gularsson) [reply src-srv/utils/CollaborationServer.ts:313]: Left behind, will remove

File: src-srv/utils/CollaborationServer.ts (hunk 1 above)
Comment (gularsson) [reply src-srv/utils/CollaborationServer.ts:308]: Left behind, will remove

**Review by dannelundqvist — APPROVED** [review]

**Pull request merged**